pkg/utils/status.go

```diff
@@ -341,50 +364,67 @@ func calculateState(ctx context.Context, instance *csmv1.ContainerStorageModule,
 	controllerStatus := newStatus.ControllerStatus
 
 	newStatus.State = constants.Failed
-	log.Infof("deployment controllerReplicas [%s]", controllerReplicas)
-	log.Infof("deployment controllerStatus.Available [%s]", controllerStatus.Available)
+	//log.Infof("deployment controllerReplicas [%s]", controllerReplicas)
+	//log.Infof("deployment controllerStatus.Available [%s]", controllerStatus.Available)
 
-	log.Infof("daemonset expected [%d]", expected)
-	log.Infof("daemonset nodeStatus.Available [%s]", nodeStatus.Available)
+	//log.Infof("daemonset expected [%d]", expected)
+	//log.Infof("daemonset nodeStatus.Available [%s]", nodeStatus.Available)
 
-	if instance.GetName() == "application-mobility" {
-		modrunning, err := statusForAppMob(ctx, instance, r, newStatus)
-		if err != nil {
-			log.Infof("statusForAppMob err msg [%s]", err.Error())
+	for _, m := range instance.Spec.Modules {
+		if(m.Name =="application-mobility" && m.Enabled) {
+			appEnabled = true
+			appRunning, err = statusForAppMob(ctx, instance, r, newStatus)
+			if err != nil {
+				log.Infof("status for Application-Mobility err msg [%s]", err.Error())
```

Comment:
@abhi16394 Should these be error statements ?

Reply:
log comments should be fine